src/runtime/BigInt.cpp

```diff
@@ -31,7 +31,9 @@ BigIntData::~BigIntData()
 
 BigIntData::BigIntData(BigIntData&& src)
```

Comment:
One more thing about this constructor.
It seems that this constructor is nowhere used.
Would you check once more?

Reply:
You are right. it is used by nowhere. but I want to keep it
because moving bf_t data is little trickly

Reply:
Oh, I see.